src/_components/StudentsProjectTeam.jsx

```diff
@@ -0,0 +1,52 @@
+export default function StudentsProjectTeam({ comp }) {
+  return (
+    <section className="grow flex flex-col lg:flex-row lg:items-center lg:min-h-[42rem] relative overflow-hidden lg:pb-32">
+      <div className="lg:w-7/12 lg:pl-40 lg:pr-20 text-negative max-lg:w-full max-lg:px-8 max-lg:flex max-lg:flex-col max-lg:justify-start max-large:items-start">
+        <h2 className="font-semibold lg:text-2xl text-5xl text-black">
+          Project Teams
+        </h2>
+        <p className="bg-[#E9415A] px-2 py-1 max-w-max my-3 rounded-md text-white lg:text-xs text-lg font-medium">
+          APPLICATION REQUIRED
+        </p>
+        <p className="text-black">
+          <strong>Purpose:</strong> Work directly on projects with non-profit
+          organizations (NPOs)!
+        </p>
+        <p className="text-black">
+          <strong>Goal:</strong> Help deliver requested software solutions to
+          NPOs and contribute to our mission of creating tech for social good!
+        </p>
+        <p className="text-black">
+          <strong>Requirements:</strong> Project teams are year long commitments
+          depending on the scope of the project. We are looking for{" "}
+          <strong className="text-[#E9415A]">
+            technical leads, project managers, developers, and designers
+          </strong>{" "}
+          who can engage and collaborate within this time frame while also
+          meeting deadlines punctually.
+        </p>
+        <p className="text-black">
+          Have any questions? Reach out to our VPs of Projects:
+        </p>
+        <comp.EmailLink
```

Comment:
There were some updates to the designs, we are switching to the blueprint email: "sit.blueprint@gmail.com" instead of individual emails for E-Board members